pmm-app/src/pmm-qan/panel/qan.constants.ts

```diff
@@ -727,7 +722,7 @@ export const METRIC_CATALOGUE = {
   },
   rows_read: {
     humanizeName: 'Bytes Read',
```

Comment:
@lunaticusgreen  ptal, this looks like a typo in name. should this be  Rows Read?

Reply:
Yes, looks os